run_unittests.py

```diff
@@ -1048,7 +1048,7 @@ def test_compiler_detection(self):
         ar = mesonbuild.linkers.ArLinker
         lib = mesonbuild.linkers.VisualStudioLinker
         langs = [('c', 'CC'), ('cpp', 'CXX')]
-        if not is_windows():
+        if not (is_windows() or is_freebsd()):
```

Comment:
I can use clang from FreeBSD base to build the archivers/unarchiver port, which is an archive extracting program written in Objective-C. What is the "separate binary and marked broken" compiler you use here?

Reply:
The guidance I got was to use lang/obj, which is neither GCC nor Clang based, and is marked broken in ports. I think this is an actual bug in meson because `cc prog.m -o prog` does work. So back to the drawing board on that one.

Reply:
Do you mean the lang/libobjc2 port? It is only marked broken on 2 tier-2 platforms.